Source/WebCore/dom/Position.h

```diff
@@ -199,6 +199,8 @@ class Position {
     // FIXME: Combine this function with operator==
     bool equals(const Position&) const;
 
+    WEBCORE_EXPORT static Node* getCommonAncestorForPositions(const Position&, const Position&);
```

Comment:
Perhaps we could just `WEBCORE_EXPORT` `commonInclusiveAncestor` instead of introducing a new method?